src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -0,0 +1,343 @@
+// ------------------------------------------------------------------------------
```

Comment:
Is this just moving from another place? If so, why did Git treat this as a new file. It should be able to handle move operations.